src/gui/editors/AutomationEditor.cpp

```diff
@@ -1302,8 +1145,8 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 		{
 			if( ( barCount + leftBars )  % 2 != 0 )
 			{
-				p.fillRect( x - m_currentPosition * zoomFactor / timeSignature, TOP_MARGIN, m_ppb,
-					height() - ( SCROLLBAR_SIZE + TOP_MARGIN ), backgroundShade() );
+				p.fillRect(x - m_currentPosition * zoomFactor / timeSignature, TOP_MARGIN, m_ppb,
+					height() - (SCROLLBAR_SIZE + TOP_MARGIN), m_backgroundShade);
```

Comment:
Not necessary to change, but my preference is if we have to split the line during an argument list, we should just split every argument to its own line. It makes it easier to find what goes into each argument (and helps if you don't have an IDE to give you argument hints).
```suggestion
				p.fillRect(
					x - m_currentPosition * zoomFactor / timeSignature,
					TOP_MARGIN,
					m_ppb,
					height() - (SCROLLBAR_SIZE + TOP_MARGIN),
					m_backgroundShade
				);
```